apps/prairielearn/src/tests/instructorQuestions.test.ts

```diff
@@ -109,32 +110,15 @@ describe('Instructor questions', function () {
       assert.isArray(questionData);
       questionData.forEach((question) => assert.isObject(question));
     });
-    it('should include addNumbers question', function () {
-      elemList = questionData.filter((question) => idsEqual(question.id, addNumbers.id));
-      assert.lengthOf(elemList, 1);
-      assert.equal(addNumbers.qid, elemList[0].qid);
-      assert.equal(addNumbers.title, elemList[0].title);
-    });
-    it('should include addVectors question', function () {
-      elemList = questionData.filter((question) => idsEqual(question.id, addVectors.id));
-      assert.lengthOf(elemList, 1);
-      assert.equal(addVectors.qid, elemList[0].qid);
-      assert.equal(addVectors.title, elemList[0].title);
-    });
-    it('should include downloadFile question', function () {
-      elemList = questionData.filter((question) => idsEqual(question.id, downloadFile.id));
-      assert.lengthOf(elemList, 1);
-      assert.equal(downloadFile.qid, elemList[0].qid);
-      assert.equal(downloadFile.title, elemList[0].title);
-    });
-    it('should include differentiatePolynomial question', function () {
-      elemList = questionData.filter((question) =>
-        idsEqual(question.id, differentiatePolynomial.id),
-      );
-      assert.lengthOf(elemList, 1);
-      assert.equal(differentiatePolynomial.qid, elemList[0].qid);
-      assert.equal(differentiatePolynomial.title, elemList[0].title);
-    });
+
+    for (const testQuestion of testQuestions) {
+      it(`should include ${testQuestion.qid} question`, function () {
+        elemList = questionData.filter((question) => idsEqual(question.id, testQuestion.id));
```

Comment:
Does `elemList` need to be declared in a higher scope? AFAICT there are two tests in which it's used and they don't (and in fact shouldn't) share state with each other.

Reply:
fixed